index.html

```diff
@@ -299,16 +299,62 @@ <h1>Sponsors</h1>
 
     <!--=================Card 6 Content Slide Navigation============-->
     <div id="myNav6" class="overlay6 tabcontent">
-
         <div class="flex-container">
             <a href="javascript:void(0)" class="closebtn" onclick="closeNav('myNav6')">&times;</a>
             <div class="image_card6"></div>
-            <div>
-                <h1>Project Showcase</h1>
-                <h3>More Information Coming Soon!</h3>
-            </div>
+            <h1>Project Showcase</h1>
+            <p>Here's a collection of the projects that were demoed from <b>UWB Hacks</b>.
+               Listed by award, then alphabetically. Descriptions provided by groups.</p>
+            <h2> Student Favorite </h2>
+            <div class = "project">
+              <div class="flip-card">
+                  <div class="flip-card-inner">
+                      <div class="flip-card-front">
+                        <img class = "project-image" src="./images/cassette.jpg" alt="student-favorite">
+                      </div>
+                      <div class="flip-card-back">
+                        <h2> Project name </h2>
+                        <h4> Team Name</h4>
+                        <p> Description about project </p>
+                      </div>
+                   </div>
+               </div>
+           </div>
+
+           <h2> Best Aesthetics </h2>
+           <div class = "project">
+             <div class="flip-card">
+                 <div class="flip-card-inner">
+                     <div class="flip-card-front">
+                       <img class = "project-image" src="./images/cassette.jpg" alt="student-favorite">
+                     </div>
+                     <div class="flip-card-back">
+                       <h2> Project name </h2>
+                       <h4> Team Name</h4>
+                       <p> Description about project </p>
+                     </div>
+                  </div>
+              </div>
+          </div>
+
+          <h2> Best Functionality</h2>
+          <div class = "project">
+            <div class="flip-card">
+                <div class="flip-card-inner">
+                    <div class="flip-card-front">
+                      <img class = "project-image" src="./images/cassette.jpg" alt="student-favorite">
+                    </div>
+                    <div class="flip-card-back">
+                      <h2> Project name </h2>
+                      <h4> Team Name</h4>
+                      <p> Description about project </p>
+                    </div>
+                 </div>
+             </div>
+         </div>
+
```

Comment:
End commented content here
```suggestion
    -->
```